docs/src/components/Layout/GlobalNav/components/AmplifyNavLink.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, Text, Button, VisuallyHidden } from '@aws-amplify/ui-react';
+import styles from '../GlobalNav.module.scss';
+import { AmplifyLogo, ChevronIcon } from './icons';
+
+export function AmplifyNavLink({ currentSite, isCollapsed, setIsCollapsed }) {
+  const chevronRotation = isCollapsed ? '0' : '180';
+  return (
+    <Flex justifyContent={'space-between'} className={styles['logo-container']}>
+      <Flex gap="xs" alignItems="center" direction="row">
+        <AmplifyLogo />
+        <Text className={styles['dev-center-logo']}>
+          <span style={{ fontWeight: '400' }}>Amplify</span>{' '}
+          <span style={{ fontWeight: '300' }}>Dev Center</span>
+          <span className={styles['mobile-only']}>
+            {' '}
+            <ChevronIcon rotateDeg="270" /> {currentSite}
+          </span>
+        </Text>
+      </Flex>
+
+      <Button
+        aria-expanded={!isCollapsed}
+        aria-controls="mobile-nav-links"
+        border="none"
+        backgroundColor="transparent"
+        className={`${styles['mobile-only']} ${styles['nav-expander']}`}
+        onClick={() => {
+          setIsCollapsed(!isCollapsed);
+        }}
+      >
+        <VisuallyHidden>
+          {isCollapsed ? 'Open Dev Center menu' : 'Close Dev Center menu'}
```

Comment:
Why do we need to add these and hide them? Should we use a `alt` in image instead?

Reply:
This is for screen readers and I think I didn't add it to the image, because I didn't want screen readers to really "know" about the chevron icon (which is why it has `aria-hidden="true"`)

Ideally screen readers don't need to know about the chevron icon as an actual image, but just need to know that this button exists and that what it reads out is based on this `VisuallyHidden` text

Reply:
I see. Thanks. #TIL